docs/content/guides/developer/app-examples/tic-tac-toe.mdx

```diff
@@ -2,354 +2,336 @@
 title: Tic-Tac-Toe
 ---
 
-This guide covers three different implementations of the game tic-tac-toe on Sui. The first example utilizes a centralized admin that marks the board on the users’ behalf. The second example utilizes a shared object that both users can mutate. And the third example utilizes a multisig, where instead of sharing the game board, it's in a 1-of-2 multisig of both users’ accounts. This guide compares and contrasts the design philosophies behind the three different games, as well as the pros and cons of each.
+This guide covers three different implementations of the game tic-tac-toe on Sui. The first example utilizes a centralized admin that marks the board on the users’ behalf. The second example utilizes a shared object that both users can mutate. And the third example utilizes a multisig, where instead of sharing the game board, it's in a 1-of-2 multisig of both users’ accounts.
 
-## tic_tac_toe.move
+## owned.move
 
 In this first example of tic-tac-toe, the game object, including the game board, is controlled by a game admin.
 
```

Comment:
Any reason not to use the plugin?
{@inject: examples/tic-tac-toe/move/sources/owned.move#struct=Game noComments}